### PR TITLE
Update Reader SDK version to latest one on both platforms.

### LINKED ISSUE #56: Module 'SquareReaderSDK' not found
I am trying to compile the example, and I get a Parse Issue: Module 'SquareReaderSDK' not found

Error Log:
    /Users/lane/Documents/reader-sdk-flutter-plugin/ios/Classes/Converters/SQRDT
    enderCashDetails+FlutterReaderSDKAdditions.m:17:
    /Users/lane/Documents/reader-sdk-flutter-plugin/ios/Classes/Converters/SQRDT
    enderCashDetails+FlutterReaderSDKAdditions.h:17:9: fatal error: module
    'SquareReaderSDK' not found
    @import SquareReaderSDK;
     ~~~~~~~^~~~~~~~~~~~~~~
    1 error generated.


### To Reproduce
Steps to reproduce the issue:
1. git clone git@github.com:square/reader-sdk-flutter-plugin.git
2. cd reader-sdk-flutter-plugin/example
3. rm ios/PodFile
3. flutter pub get
4. edit ios/PodFile to set platform to 11.1
4. flutter build iOS


**Environment (please complete the following information):**
platform: iOS
macOS: 11.2.2
Flutter 2.1.0-11.0.pre.145 • channel master •
https://github.com/flutter/flutter.git
Framework • revision a0ba646408 (22 hours ago) • 2021-03-05 22:09:03 -0500
Engine • revision f751d04fa0
Tools • Dart 2.13.0 (build 2.13.0-107.0.dev)
Reader SDK:

### PR DESCRIPTION
**Pull request by shivamsaviant**
## Summary

- Upgrade Reader SDK version to `1.4.4`
- Update `targetSdkVersion` to `29`
- Update document 

## Related issues

Fix #56 

## Changelog

* Upgrade Reader SDK version to `1.4.4`
* Update `targetSdkVersion` to `29`
* Update document

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -6,36 +6,36 @@ PODS:
   - MTBBarcodeScanner (5.0.8)
   - permission_handler (3.3.0):
     - Flutter
-  - square_reader_sdk (2.0.0):
+  - square_reader_sdk (2.1.0):
```
Comment (jguze) [inline example/ios/Podfile.lock:9]: Why was this bumped to 2.1?

File: example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>BuildSystemType</key>
```
Comment (jguze) [inline example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:5]: Is it necessary to check in these preferences?

**Review by jguze — APPROVED** [review]
Please just verify my other two comments, but otherwise it looks good to me:

- [ ] Double check open comments

**Comment by hukid** [comment]
- [ ] please fix failed travis job.

**Review by hukid — APPROVED** [review]

File: example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (hunk 2 above)
Comment (shivamsaviant) [reply example/ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings:5]: We can revert this to original.

File: example/ios/Podfile.lock (hunk 1 above)
Comment (shivamsaviant) [reply example/ios/Podfile.lock:9]: We have increased the version [here](https://github.com/square/reader-sdk-flutter-plugin/blob/2a70b0ed0836a389c39ded2e94e56041c180af86/ios/square_reader_sdk.podspec#L6) in square_reader_sdk.podspec, hence it takes version from there.

**Pull request merged**